docs/kubernetes/kctlr-create-service-vs.rst

```diff
@@ -101,7 +105,14 @@ Verify changes on the BIG-IP system
 
 .. include:: /_static/reuse/verify-bigip-objects.rst
 
-What's next
------------
 
-- :ref:`kctlr-manage-bigip-objects`
\ No newline at end of file
+.. _kctlr-downed-services:
```

Comment:
Removing a specific Service will remove the objects from the BIG-IP as well as removing the F5 objects - this should be the same for Ingress and Route.

Removing the CC itself will do something to this effect though. It doesn't delete anything on shutdown. So you can remove the CC and conduct tests and make changes. And when bringing it back up have those changes made to the system or repaired from the state of the orchestration system.